contracts-periphery/Makefile

```diff
@@ -6,8 +7,11 @@ else
   INSTALL_CMD := forge install
 endif
 
+forkBlock = 14800081
+
 build :; forge build
 clean :; forge clean
 install :; $(INSTALL_CMD)
-snapshot :; forge snapshot --match-path test/UmbraBatchSend.gas.t.sol
-test :; forge test
\ No newline at end of file
+snapshot :; forge snapshot -f ${ETH_RPC_URL} --fork-block-number ${forkBlock} --match-path \*.gas.t.sol
+test :; forge test --match-path test/UmbraBatchSend.t.sol && make test-hook
```

Comment:
Note change: `make test` no longer tests `UmbraBatchSend.gas.t.sol`, should I add `&& forge test --match-path test/gas/UmbraBatchSend.gas.t.sol` to the end of this line?